nautobot/ipam/templates/ipam/role.html

```diff
@@ -87,11 +29,9 @@ <h1>{{ object }}</h1>
                 </tr>
             </table>
         </div>
-        {% include 'inc/custom_fields_panel.html' %}
-        {% include 'inc/relationships_panel.html' %}
-        {% plugin_left_page object %}
-	</div>
-	<div class="col-md-8">
+{% endblock content_left_page %}
+
+{% block content_right_page %}
```

Comment:
Consider moving to `content_full_width_page`:

![image](https://user-images.githubusercontent.com/5603551/127335499-19e7684d-e2d0-4e82-8e43-bb2f7410b079.png)